hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenGetInfoHandler.java

```diff
@@ -58,13 +80,117 @@ public Response createEmptyResponse(@NonNull final ResponseHeader header) {
     @Override
     public void validate(@NonNull final QueryContext context) throws PreCheckException {
         requireNonNull(context);
-        throw new UnsupportedOperationException("Not implemented");
+        final var query = context.query();
+        final var tokenStore = context.createStore(ReadableTokenStore.class);
+        final var op = query.tokenGetInfoOrThrow();
+        if (op.hasToken()) {
```

Comment:
You could do `validateTruePreCheck(op.hasToken, INVALID_TOKEN_ID)`

Reply:
updated, thanks.